pkg/platform/github.go

```diff
@@ -398,6 +398,68 @@ func (g *GitHub) withRetries(ctx context.Context, retryFunc retry.RetryFunc) err
 	return nil
 }
 
+// ModifierContent returns the pull request body as the content to parse modifiers
+// from.
+func (g *GitHub) ModifierContent(ctx context.Context) (string, error) {
```

Comment:
This function uses the github client instead of wrapper functions within the struct like main does because they dont exist in this branch.